examples/twittermap/web/public/javascripts/lib/hashmap.js

```diff
@@ -0,0 +1,210 @@
+/**

```

Comment:
this is not the correct way of adding a dependency. you can take a look at this [line](https://github.com/ISG-ICS/cloudberry/blob/master/examples/twittermap/project/dependencies.scala#L63) to add a dependency. @vidhya567  just in case you have a minute to help a little on adding dependencies.

Reply:
Yeah I saw a bower module for this library in webinar. she can you it

Reply:
@dharini-s can you communicate with @vidhya567 about how to add the dependency?

Reply:
Sure, @JavierJia I'll check with @vidhya567.